webapp-django/crashstats/signature/static/signature/js/signature_tab_correlations.js

```diff
@@ -182,6 +182,18 @@ SignatureReport.CorrelationsTab.prototype.onAjaxSuccess = function () {
                 ['core-counts', 'interesting-addons', 'interesting-modules']
             );
         });
+    }
+
+    function load() {
+        var channel = $('#correlations_channel').val();
+        var product = $('#correlations_product').val();
+        var signature = $('#correlations_signature').val();
```

Comment:
You don't need to use a hidden input, the signature is already accessible with ``SignatureReport.signature``.